pynucastro/networks/python_network.py

```diff
@@ -6,6 +6,21 @@
 from pynucastro.networks.rate_collection import RateCollection
 from pynucastro.rates.rate import ApproximateRate
 
+TABLEINDEX_DEF = """
```

Comment:
You can switch the outer quotes to `'''` so you don't have to escape the inner ones

Reply:
done